sycl/test-e2e/AddressSanitizer/common/options-invalid.cpp

```diff
@@ -0,0 +1,39 @@
+// REQUIRES: linux
```

Comment:
This is an incorrect location for this test. Functionality that is being tested resides in Unified Runtime and therefore it should be tested there